lib/eco/incparser/astree.py

```diff
@@ -262,6 +262,18 @@ def load(self, version):
                 return
             version -= 1
 
+    def get_attr(self, attr, version):
+        if version is None:
+            return self.__getattribute__(attr)
+        version = int(version)
+        while version >= 0:
+            try:
+                return self.log[(attr, version)]
+            except KeyError:
+                version -= 1
+        raise Exception("Warning: reached zero", self, self.log)
+        return None
```

Comment:
Ah, yes you are right. I'm actually not even sure what I want to do here, as in if I even want an exception here. I guess I go with a exception for now. Do you have a suggestion for a better exception?